tests/e2e/e2e.sh

```diff
@@ -17,83 +17,57 @@
 set -eux
 
 export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
+ROOT=$(cd ../../../; pwd)
 
-function cleanup_kind_cluster() {
-    kind export logs --name istio-testing "${ARTIFACTS}/kind"
-    kind delete cluster --name=istio-testing
-}
-
-function setup_kind_cluster() {
-  # Delete any previous e2e KinD cluster
-  echo "Deleting previous KinD cluster with name=istio-testing"
-  if ! (kind delete cluster --name=istio-testing) > /dev/null; then
-    echo "No existing kind cluster with name istio-testing. Continue..."
-  fi
-
-  trap cleanup_kind_cluster EXIT
-
-  # Create KinD cluster
-  if ! (kind create cluster --name=istio-testing --loglevel debug --retain --wait 30s); then
-    echo "Could not setup KinD environment. Something wrong with KinD setup. Exporting logs."
-    exit 1
-  fi
-
-  KUBECONFIG="$(kind get kubeconfig-path --name="istio-testing")"
-  export KUBECONFIG
-
-cat <<EOF > metallb-config.yaml
```

Comment:
getting metallb to work on Linux was hard, but I can revive this work later if need be.  Please open a separate PR and place a hold on it to add metlalb back to the system and I can take on reimplementation from there within the PR queue.